build.gradle

```diff
@@ -43,6 +44,26 @@ groovydoc {
     link 'https://docs.groovy-lang.org/latest/html/gapi/', 'groovy.', 'org.codehaus.groovy.'
 }
 
+//def signingKey = System.getenv("SIGNING_KEY")
+def signingKey = ""
+//def signingPassword = System.getenv("SIGNING_PASSWORD")
+def signingPassword = ""
```

Comment:
I guess we need to switch back to use `System.getenv`?

Reply:
Oops, yes. That was from me testing locally that a blank variable was causing the problems. I will update shortly